package.json

```diff
@@ -10,7 +10,8 @@
     "check-ts": "tsc --noEmit && tsc --project test/tsconfig.json --noEmit",
     "check-lint": "tsc --noEmit && eslint 'src/**/*.ts' 'test/**/*.ts'",
     "check-format": "prettier -c 'src/**/*.ts' 'test/**/*.ts'",
-    "check-spell": "cspell 'src/**/*ts' 'test/**/*ts'"
+    "check-spell": "cspell 'src/**/*ts' 'test/**/*ts'",
+    "format": "prettier -w './**/*.{ts,md,js,json}'"
```

Comment:
If you want to add the `format` command you should be consistent with the `check-format`, we verify only `ts`

Reply:
left only `ts` for the format